.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,205 @@
+
+* Skip step 2 of the contribution process.
+* In step 3 of the contribution process, clone directly the original repo:
+  ```
+  git clone https://github.com/SysBioChalmers/yeast-GEM.git
+  ```
+
+Follow all other steps in the same way. Also, when creating your pull request (or after you have created it):
+* Choose 1 or more members of the team (ideally with knowledge on the pull request) as reviewers. Note that the person making the pull request and the reviewer _cannot_ be the same person.
+* Assign appropriate [labels](https://github.com/SysBioChalmers/yeast-GEM/issues/labels).
+* Assign the pull request to one of the available [projects](https://github.com/SysBioChalmers/yeast-GEM/projects), if applicable.
+	
+### Reviewing pull requests
+
+Every pull request must be approved by at least one reviewer before it can be merged. When reviewing someone else's pull request, keep in mind the following aspects:
+* **Compatibility:** First of all, make sure that the model is still compatible with the loading/saving wrappers (`loadYeastModel.m` & `saveYeastModel.m`) and that no errors appear. Check also that [`dependencies.txt`](https://github.com/SysBioChalmers/yeast-GEM/blob/master/ModelFiles/dependencies.txt) does not change in any unexpected ways (e.g. an "unknown" toolbox version). Finally, ensure that the SBML fields `model metaid`, `model id` and `model name` never change, as if they change it would create a conflict in the next release.
+* **Documentation:** Every change should be justified with a reference/link/argument. This can be provided as data in `/ComplementaryData`, or directly as a comment in the pull request.
+* **Reproducibility:** If there are any added scripts, make sure that if you run them, the model gets updated from how it was in `devel` to how it is in the pull request. For this, you may _locally_ switch to the corresponding branch, replace the `.xml` file with the one from `devel` before the changes, and run the associated scripts. Remember to stash any undesired changes afterwards.
+* **Style:** Ensure that the changes to the model are compliant with the model's rxn/met/gene naming conventions (when unsure, take a look at a similar field in the model). Also, make sure that scripts have a compliant style, and datasets are straight-forward to understand.
+* When commenting in the review, please comply with our [code of conduct](https://github.com/SysBioChalmers/yeast-GEM/blob/master/.github/CODE_OF_CONDUCT.md).
+* Avoid vague comments and try to be as explicit as possible (e.g.: _"please change this"_ instead of _"this could be changed"_).
```

Comment:
To be honest, the examples that you give here seem vague..

Reply:
these were not intended to be the whole messages, and I have clarified that in the text